Orange/widgets/evaluate/tests/common.py

```diff
@@ -0,0 +1,23 @@
+from Orange import classification, evaluation
+from Orange.data import Table
+
+
+def color_palette_test():
+    """
+    Now works with more than 9 evaluation results.
+    GH-2394
+    """
+    data = Table("iris")
+    learners = [
+        classification.MajorityLearner(),
+        classification.LogisticRegressionLearner(),
+        classification.TreeLearner(),
+        classification.SVMLearner(),
+        classification.KNNLearner(),
+        classification.CN2Learner(),
+        classification.SGDClassificationLearner(),
+        classification.RandomForestLearner(),
+        classification.NaiveBayesLearner(),
+        classification.SGDClassificationLearner()
+    ]
+    return evaluation.CrossValidation(data, learners, k=2, store_data=True)
```

Comment:
Rename `common.py` to `base.py`. Instead of function `color_palette_test`, `base.py` should include a class derived from `WidgetTest` with a method `test_many_evaluation_results`. `TestOWROCAnalysis`, `TestOWCalibrationPlot` and `TestOWLiftCurve` are then derived from this class.